data.tf

```diff
@@ -8,6 +8,7 @@ data "aws_iam_policy_document" "dynamo_firehose_adapter" {
     ]
     resources = [
       var.dynamodb_stream_arn,
+      var.tags
```

Comment:
This is wrong.
This is a policy document. And resources are AWS resources that the IAM policy is permitting the above actions